collect_app/build.gradle

```diff
@@ -115,7 +111,7 @@ android {
             minifyEnabled(true)
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
             resValue("string", "GOOGLE_MAPS_API_KEY", googleMapsApiKey)
-            buildConfigField 'String', "MAPBOX_ACCESS_TOKEN", '"' + mapboxToken + '"'
+            resValue("string", "mapbox_access_token", mapboxAccessToken)
```

Comment:
This works out of the box. We generate a string named `mapbox_access_token` with a valid access token and mapbox takes it automatically to setup everything that it needs.